### PR TITLE
Add tags

### PR DESCRIPTION
**Pull request by artamonovkirill**
Following https://en.wikipedia.org/wiki/Apache_Groovy

Added "declarative" (`collect`, `inject`, `findAll`, ...)  and "functional" (see https://en.wikipedia.org/wiki/Apache_Groovy#Functional_programming) styles.

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -784,5 +784,23 @@
...
+    "paradigm/declarative",
+    "paradigm/functional",
+    "paradigm/imperative",
+    "paradigm/object_oriented",
+    "typing/static",
+    "typing/dynamic",
+    "execution_mode/compiled",
+    "paradigm/declarative",
+    "paradigm/functional",
+    "paradigm/object_oriented",
```
Comment (ErikSchierboom) [inline config.json:797]: This seems to be a duplicate of the first couple of lines

**Review by ErikSchierboom — APPROVED** [review]
One small nit

**Review by ErikSchierboom — APPROVED** [review]

**Comment by artamonovkirill** [comment]
@ErikSchierboom how to proceed with this PR?

It has the same 2 jobs failing as on main. At the moment I'm too unfamiliar with the project setup to solve those.

**Comment by ErikSchierboom** [comment]
@artamonovkirill Those build failures are unrelated to this PR, so I'm gonna merge this. Thanks!

**Pull request merged**